### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by H34D**
## Description

- made networks and own bullet point in the TOC
- renamed Nile Beta network to Nile testnet
- deprecated kovan testnet
- flipped the order of ethereum mainnet and pacific mainnet, pacific is now topmost
- added a reference to docs/contracts to the documentation section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -257,6 +268,7 @@ See [RELEASE_PROCESS.md](RELEASE_PROCESS.md)
 ## Documentation
 
 * [Main Documentation](doc/)
+* [Contracts](doc/contracts/)
```
Comment (0x3bfc) [inline README.md]: What is the difference between `Main Documentation` and `contracts` ?  I think they point to the same content but in a different way ?

Comment (H34D) [reply README.md]: Is that so?

right now `doc` contains three things:

- The contract documentation incl. a reference to them
- the packaging documentation
- the upgrade documentation

Comment (0x3bfc) [reply README.md]: I see, so we should update this document https://github.com/oceanprotocol/keeper-contracts/tree/715d677886bd1f9fa855f4aebcabd45215f7b2d6/doc to point to three files in the table of content

Comment (H34D) [reply README.md]: upated

**Pull request merged**